packages/python-pypdf/PKGBUILD

```
@@ -0,0 +1,40 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-pypdf
+_pkgname=${pkgname#python-}
+pkgver=3.2.0
+pkgrel=1
+pkgdesc='Python library for manipulating pages of PDF files (v3+)'
+arch=('any')
+url="https://github.com/py-pdf/$_pkgname"
```

```suggestion
url="https://github.com/py-pdf/py-pdf"
```

We have a convention not putting variables in url.